vendor/github.com/vmware/govmomi/object/datastore_file.go

```diff
@@ -344,9 +345,10 @@ func (f *DatastoreFile) TailFunc(lines int, include func(line int, message strin
 }
 
 type followDatastoreFile struct {
```

Comment:
Is this a change in upstream govmomi? or should this also be in a PR for govmomi? @dougm